pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { GetServerSideProps } from "next";
+
+import React, { useState } from "react";
+import Link from "next/link";
+import { Formik } from "formik";
+import { useTranslation } from "next-i18next";
+import * as Yup from "yup";
+import axios from "axios";
+import { logMessage } from "@lib/logger";
+import {
+  Button,
+  TextInput,
+  Label,
+  Alert,
+  ErrorListItem,
+  MultipleChoiceGroup,
+  TextArea,
+  Description,
+} from "@components/forms";
+import { StyledLink } from "@components/globals/StyledLink/StyledLink";
+
+export default function Contactus() {
+  const { t, i18n } = useTranslation(["form-builder", "common"]);
+  const [errorState, setErrorState] = useState({ message: "" });
+  const [submitting, setSubmitting] = useState(false);
+  const [isSuccessScreen, setIsSuccessScreen] = useState(false);
+
+  const handleRequest = async (
+    name: string,
+    email: string,
+    request: string,
+    description: string
+  ) => {
+    const type = "contactus";
+    return await axios({
+      url: "/api/request/help",
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      data: { type, name, email, request, description },
+      timeout: process.env.NODE_ENV === "production" ? 60000 : 0,
+    }).catch((err) => {
+      logMessage.error(err);
+      setErrorState({ message: t("submissionError") });
+    });
+  };
+
+  const validationSchema = Yup.object().shape({
+    name: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .max(50, t("signUpRegistration.fields.name.error.maxLength")),
+    email: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .email(t("input-validation.email", { ns: "common" })),
+    request: Yup.string().required(t("input-validation.required", { ns: "common" })),
+    description: Yup.string().required(t("input-validation.required", { ns: "common" })),
+  });
+
+  return (
+    <div aria-live="polite">
+      {!isSuccessScreen && (
+        <Formik
+          initialValues={{ name: "", email: "", request: "", description: "" }}
+          onSubmit={async ({ name, email, request, description }) => {
+            setSubmitting(true);
+            try {
```

Comment:
Not exactly sure why a try catch would be needed here. The handleRequest function already handles the error and does not bubble it up. Not sure its good practice to throw a new error to handle when a bad response is returned. Furthermore why not just do all of this in handleRequest ? Otherwise an if statement should suffice on the response status here